src/style.css

```diff
@@ -0,0 +1,117 @@
+*,
+::after,
+::before {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 0 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__imac-img {
+  width: 160px;
+  height: 134px;
+  margin: 30px auto 45px;
+}
+
+.card__name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
+.card__code {
+  margin-top: 5px;
+  font-size: 11px;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__star {
+  width: 15px;
+  height: 15px;
+  margin-left: 4px;
+  background: url(images/star.svg) no-repeat center;
+}
+
+.card__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background: url("images/star-active.svg") no-repeat center;
+}
+
+.card__rating {
+  margin-top: 15px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__review {
+  font-size: 11px;
+  margin-bottom: 1px;
+
+}
+
+.card__price {
+  margin-top: 20px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__price-text {
+  font-size: 11px;
+  margin-bottom: 1px;
+}
+
+.card__price-number {
+  font-weight: bold;
+  font-size: 16px;
+}
+
+.card__buy {
+  margin-top: 20px;
+  width: 170px;
+  height: 40px;
+
+  background-color: #00acdc;
+  border-radius: 5px;
+  color: #fff;
+  font-size: 14px;
+  font-weight: bold;
+
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+  line-height: 40px;
+}
+
+.card__buy:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
```

Comment:
don't add border to the button on hover. Add it to initial button state and make transparent

Checklist